app/views/includes/ptaHeader.scala.html

```diff
@@ -45,6 +45,19 @@
 
     </div>
 }
+
+
+<div class="grid-row">
+    <div class="column-full">
+       <div class="print-banner">
+        <div class="hmrc-logo"><img src="/check-income-tax/assets/images/HMRC-logo.png" alt='HM Revenue & Customs'></div>
```

Comment:
should the message come from a label?

Reply:
do you mean the alt attribute  - if so  then I think we should leave as is - because its redundant in this case (ie because its for printing only and not screen or screen reader use